cypress/support/step-definitions/rainbow-steps.js

```diff
@@ -0,0 +1,35 @@
+import {
+  titlePreview, nameAndLabelPreview, firstBitPreview, tooltip,
+} from '../../locators/rainbow';
+import { getKnobsInput } from '../../locators';
+
+When('I put {string} json to {string} input field', (json, inputFieldName) => {
+  cy.fixture(json).then(($json) => {
+    getKnobsInput(inputFieldName).clear()
+      .then($selector => $selector.val(JSON.stringify($json))).type(' ');
+  });
+});
+Then('rainbow title is {string}', (title) => {
+  cy.wait(500); // required because element exists before change
+  titlePreview().should('have.text', title);
+});
+
+Then('name and label is set from {string} json', (json) => {
```

Comment:
in my opinion we could combine below 3 steps into 1 using switch expression. It will be more readable;